src/method-doc.js

```diff
@@ -145,10 +145,15 @@ function returnType(json) {
 }
 
 function formatDocType(json, type, defaultValue) {
+
   if (json.formatTypes === false) {
     return `<dfn class="doc-type">${escapeHtml(type)}</dfn>`;
   }
   else {
+    if (type.includes(' *')) {
+      type = type.slice(0, type.indexOf('*')), + type.slice(type.indexOf('*') + 1);
```

Comment:
What's going on here?
This makes `type` equal to number (`0` in this case`).
```javascript
type = type.slice(0, type.indexOf('*')), + type.slice(type.indexOf('*') + 1);
```
is basically
```javascript
t = type
type = t.slice(0, t.indexOf('*'))
type = +t.slice(t.indexOf('*') + 1)
```
So remove the comma.

You can do `type.replace(/\s*\*$/, '')` to drop the last `*` with zero or more whitespace.